scripts/images/vmware/Dockerfile.dapper

```diff
@@ -0,0 +1,83 @@
+FROM ubuntu:16.04
+# FROM arm64=aarch64/ubuntu:16.04
+
+# get the apt-cacher proxy set
+ARG APTPROXY=
+
+RUN echo "Acquire::http { Proxy \"$APTPROXY\"; };" >> /etc/apt/apt.conf.d/01proxy \
+    && cat /etc/apt/apt.conf.d/01proxy \
+    && apt-get update \
+    && apt-get install -y --no-install-recommends \
+        build-essential \
+        ca-certificates \
+        cpio \
+        curl \
+        dosfstools \
+        gccgo \
+        genisoimage \
+        gettext \
+        git \
+        isolinux \
+        less \
+        libblkid-dev \
+        libmount-dev \
+        libselinux1-dev \
+        locales \
+        module-init-tools \
+        mtools \
+        openssh-client \
+        pkg-config \
+        qemu \
+        qemu-kvm \
+        rsync \
+        sudo \
+        syslinux-common \
+        vim \
+        wget \
+        xorriso \
+        xz-utils \
+        telnet
+########## Dapper Configuration #####################
+
+ENV DAPPER_SOURCE .
+ENV DAPPER_OUTPUT dist
+ENV DAPPER_DOCKER_SOCKET true
+ENV DAPPER_RUN_ARGS --privileged
+ENV SHELL /bin/bash
+ENV DAPPER_ENV VERSION
+WORKDIR ${DAPPER_SOURCE}
+
+########## General Configuration #####################
+
+ARG DAPPER_HOST_ARCH=amd64
+ARG HOST_ARCH=${DAPPER_HOST_ARCH}
+ARG ARCH=${HOST_ARCH}
+
+ARG OS_REPO=rancher
+ARG HOSTNAME_DEFAULT=rancher
+ARG DISTRIB_ID=RancherOS
+ARG KERNEL_VERSION_amd64=4.15.9-rancher
```

Comment:
I think we do not need kernel version here

Reply:
BUILD need this arg `vmlinuz-${KERNEL_VERSION}`